nss-day-cohort-25/the-librarian-grubby-crocodiles
# Library database

Created Library array which holds objects of books.

to test run
```
git fetch --all
git checkout libraryDatabase
```

1. check database files for function to create objects, and array to hold objects
1. run server and type 'Library' into console to log out the complete library database
